src/style/style.css

```diff
@@ -1,7 +1,46 @@
 body {
   background: #123456;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
 }
 
-html, body {
+*:focus{
+  outline: none;
+}
+
+html,
+body {
   overflow: hidden;
 }
+
+.button-container {
+  display: flex;
+  flex-direction: column;
+  width: 40%;
+  position: absolute;
+  justify-items: center;
+  align-items: center;
+}
+
+.menu-btn {
+  margin: 5px;
+  background-color: transparent;
+  opacity: 0.5;
+  width: 95%;
+  padding: 20px;
+  border-radius: 10px;
+  font-size: 3rem;
+  color: white;
+}
+
```

Comment:
Maybe we could add focus to buttons when they are hovered. It creates a smoother feeling when navigating the menu with the mouse:

```css
.menu-btn:hover {
  opacity: 1;
  border-width: 5px;
  border-color: white;
}
```

Reply:
I added it as a "mouseover" event listener instead. This way we did not need to copy already existing css rules, also it does not mess with the script handling the focus (adding as css does).